src/python/WMCore/RequestManager/RequestDB/Settings/RequestStatus.py

```diff
@@ -35,19 +30,14 @@
 
 NextStatus = {
  "new" : ["new", "testing-approved", "assignment-approved", "rejected"],
- "testing-approved" : ["testing-approved", "testing", "test-failed", "aborted"],
- "testing" : ["testing", "tested", "test-failed", "aborted"],
- "tested" : ["tested", "assignment-approved","failed", "rejected", "aborted"],
- "test-failed" : ["test-failed", "testing-approved", "rejected", "aborted"],
  "assignment-approved" : ["assignment-approved", "assigned", "rejected"],
  "assigned" : ["assigned","acquired", "aborted", "failed"],
- "acquired" : ["running-open", "failed", "completed", "acquired", "aborted"],
- "running" : ["completed", "aborted", "failed", "epic-FAILED"],
- "running-open" : ["running-closed", "aborted", "failed", "epic-FAILED"],
- "running-closed" : ["force-complete", "completed", "aborted", "failed", "epic-FAILED"],
+ "acquired" : ["running-open", "completed", "acquired", "aborted"],
```

Comment:
I don't think anybody move the failed status manually. I think Ops use only reject an abort. Failed status moves automatically if the work is not pulled for a long time by workqueue, The problem is there is the case when only partial work is done and other element got stuck somewhere it still have possibility moving to failed status automatically